src/keys.ts

```diff
@@ -242,11 +242,13 @@ export const memberKeys = {
       /**
         This should hold RecycledItemData
       */
-      recycled: [...currentBaseKey, 'recycled'] as const,
-      /**
-        This should hold items that have been recycled
-      */
-      recycledItems: [...currentBaseKey, 'recycledItems'] as const,
+      allRecycled: () => [...currentBaseKey, 'recycled'] as const,
```

Comment:
Is it mandatory to be a function?

```suggestion
      allRecycled: [...currentBaseKey, 'recycled'] as const,
```